plugins/node/instrumentation-kafkajs/src/types.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Span } from '@opentelemetry/api';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import type { Message } from 'kafkajs';
```

Comment:
we should verify that these types do not leak into the public API where it might import types from `kafkajs` and fail if the package is not found in end user node_modules.

Reply:
I support this, will give us greater flexibility and robustness over time 👍

Reply:
Combined the generic version with the unified `info` argument